device/transport/http/test/_http_test.js

```diff
@@ -403,8 +377,32 @@ describe('HttpReceiver', function () {
       var received = 0;
       receiver.on('message', function () {
         received++;
-        if (received === msgCount) done();
+        if (received === msgCount) {
```

Comment:
>=== [](start = 21, length = 3)

recommend >= to catch the case where you receive more messages than expected.

Reply:
doesn't hurt to be defensive...

---
In reply to: [145245315](https://github.com/Azure/azure-iot-sdk-node/pull/109#discussion_r145245315) [](ancestors = 145245315)